src/components/Table/DataTableErrors.vue

```diff
@@ -0,0 +1,84 @@
+TODO: this isn't working as expected. Need to fix it.
+
+<script>
+  import { ref, onMounted } from "vue";
+  import DataTable from "./DataTable.vue";
+
+  export default {
+    components: { DataTable },
+    setup() {
+      const data = ref([]); // Store combined data
+      const BASE_URL = process.env.API_BASE_URL;
+
+      const columns = [
+        { title: "Title", data: "title" }, //TODO: Add link to package details page
+        { title: "Package ID", data: "identifier" },
+        { title: "Origin", data: "origin" },
+        { title: "Service Error", data: "service" }, //TODO: Add link to error details page
+        { title: "Date/Time", data: "last_modified"},
+      ];
+
+      async function fetchData() {
+        console.log("BASE_URL:", BASE_URL);
+
+        try {
+          // Fetch all events
+          console.log(`Fetching events from: ${BASE_URL}/events`);
```

Comment:
You should be able to get the failed events directly by passing an `outcome` URL param (this is documented in the README for `zodiac_backend` but it may be a little too subtle). So `${BASE_URL}/events?outcome=FAILURE`